### PR TITLE
fix(eslint): manual fix of eslint errors on ama-terasu

### PR DESCRIPTION
**Pull request by matthieu-crouzet**

### CODE REVIEW

**Review by mrednic-1A — APPROVED** [review]

**Review by vscaiceanu-1a — APPROVED** [review]
LGTM

File: packages/@ama-terasu/cli/src/helpers/module.helper.ts [hunk 1]
```diff
@@ -179,7 +184,8 @@ export const getLocalDependencies = async (): Promise<Record<string, string>> =>
  * @param options.localOnly Resolve module locally only
  * @returns list of modules to load
  */
-export const getCliModules = async (options: { localOnly: boolean } = { localOnly: false }): Promise<ModuleDiscovery[]> => {
+export const getCliModules = async (options?: { localOnly: boolean }): Promise<ModuleDiscovery[]> => {
```
Comment (kpanot) [inline packages/@ama-terasu/cli/src/helpers/module.helper.ts:187]: Why this change?
The caller is actually losing information regarding the default value :S.
So it should be kept as previously or `@default` should be specified (even if, in this case `undefined` is `falsy`, it is not the same as `false`)

Comment (matthieu-crouzet) [reply packages/@ama-terasu/cli/src/helpers/module.helper.ts:187]: The following rule was raising an error: [no-object-as-default-parameter](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v55.0.0/docs/rules/no-object-as-default-parameter.md)
I can specify the `@default` in tsdoc if you want

Comment (kpanot) [reply packages/@ama-terasu/cli/src/helpers/module.helper.ts:187]: There is no rule to enforce in in case of object used as function param?

Comment (fpaul-1A) [reply packages/@ama-terasu/cli/src/helpers/module.helper.ts:187]: I believe it already applies only for object
I would have liked that this triggered an error only when the object have multiple fields

Comment (matthieu-crouzet) [reply packages/@ama-terasu/cli/src/helpers/module.helper.ts:187]: As discussed with the team we keep this rule activated
